assets/wizards/advertising/views/placements/index.js

```diff
@@ -47,16 +51,122 @@ const getAdUnitsForSelect = adUnits => {
 	];
 };
 
+/**
+ * Whether any `sizesToCheck` size exists in `sizes`.
+ *
+ * @param {Array} sizes Array of sizes.
+ * @param {Array} sizesToCheck Array of sizes to check.
+ * @return {Boolean} Whether any size was found.
+ */
+const hasAnySize = ( sizes, sizesToCheck ) => {
+	return sizesToCheck.some( sizeToCheck => {
+		return ( sizes || [] ).find(
+			size => size[ 0 ] === sizeToCheck[ 0 ] && size[ 1 ] === sizeToCheck[ 1 ]
+		);
+	} );
+};
+
+/**
+ * Placement Control Component.
+ */
+const PlacementControl = ( {
```

Comment:
Non-blocking suggestion: does this component warrant its own module? It seems to have its own state and logic.

Reply:
Absolutely! It distributes the code nicely. 084b0cbce5c1577a39f85b55352730f95d3f3b45